### ISSUE TITLE
Gitter Conversation

### ISSUE THREAD

**Issue by evgeny-p** [DashProk1407/repoz#4]
У меня нет прав писать комментарии в соседнюю задачу. Прямо сейчас у меня открыт чат, можно обсудить там.

**Comment by DashProk1407** [comment]
Да, я закрыла ту задачу. Разобралась уже с помощью Максима Курносенко. Компилировать нужно было с -g3, так как gdb игнорировал макрос.
